src/components/modebar/index.js

```diff
@@ -97,18 +122,31 @@ proto.createButton = function (config) {
     button.setAttribute('rel', 'tooltip');
     button.className = 'modebar-btn';
 
-    if (config.attr !== undefined) button.setAttribute('data-attr', config.attr);
-    if (config.val !== undefined) button.setAttribute('data-val', config.val);
-    button.setAttribute('data-title', config.title);
+    button.setAttribute('data-title', config.title || '');
```

Comment:
yes, but only if `title===undefined` - would be nice if you could provide `''` or `null` and not get a tooltip at all.